### DISCUSSION TITLE
AWS EC2 bloqueando a porta 25

### DISCUSSION THREAD

**Discussion by thiagossampaio** [postalserver/postal#2410] [Help with using Postal]
It seems that not only AWS blocks port 25, requiring a request to support to release it, but also several server providers, as well as internet providers, block the same port.
What can we do to solve this problem since Postal cannot modify the email SENDING port?

I found several issues here on Github, many of them closed, but none with the solution to this problem.

Below is the exception thrown by Postal

**Comment by willpower232** [comment]
You will have to contact AWS (or your server provider of choice) and ask them to permit you to send email if necessary. In AWS case you also need to do this in order to set up reverse DNS for your IP address.
